### DISCUSSION TITLE
how to overwrite the default wsl cpu and mempry limits ?

### DISCUSSION THREAD

**Discussion by Kiddinglife** [rancher-sandbox/rancher-desktop#2424] [General]
Hey, the windows rancher-desktop by default use half of the CPU and memory sizes in the host machine.
Is there a way to overwrite those values? Thx.

**Comment by jandubois** [comment]
You can configure this in the `%UserProfile%\.wslconfig` file: https://docs.microsoft.com/en-us/windows/wsl/wsl-config#configuration-setting-for-wslconfig

These are global settins, as the WSL2 virtual machine is shared by all your distros (they run in different namespaces inside the same VM).
